pkg/common/unittestcommon/utils.go

```diff
@@ -217,3 +217,22 @@ func (c *FakeK8SOrchestrator) GetNodesForVolumes(ctx context.Context, volumeID [
 	nodeNames := make(map[string][]string)
 	return nodeNames
 }
+
+// GetNodeIDtoNameMap returns a map containing the nodeID to node name
+func (c *FakeK8SOrchestrator) GetNodeIDtoNameMap(ctx context.Context) map[string]string {
+	log := logger.GetLogger(ctx)
+	log.Debugf("GetNodeIDtoNameMap for FakeK8SOrchestrator not yet implemented")
+
+	nodeIDToNamesMap := make(map[string]string)
+	return nodeIDToNamesMap
+}
+
+// GetFakeAttachedVolumes returns a map of volumeIDs to a bool, which is set
+// to true if volumeID key is fake attached else false
+func (c *FakeK8SOrchestrator) GetFakeAttachedVolumes(ctx context.Context, volumeID []string) map[string]bool {
+	log := logger.GetLogger(ctx)
```

Comment:
Same as above (Line 233, 234 not needed)

Reply:
done